metal/mmtl/payload.py

```diff
@@ -49,21 +50,23 @@ def add_label_set(
             )
         elif label_list is not None:
             assert label_fn is None
-            assert isinstance(label_list, torch.Tensor)
+            assert isinstance(label_list, torch.Tensor) or isinstance(
+                label_list, np.ndarray
+            )
             new_labels = label_list
         else:
             raise ValueError("Incorrect label object type -- supply list or function")
 
-        if new_labels.dim() < 2:
-            raise Exception("New label_set must have at least two dimensions: [n, ?]")
+        # if new_labels.dim() < 2:
+        #    raise Exception("New label_set must have at least two dimensions: [n, ?]")
```

Comment:
I think we can get rid of this if we want to allow the use of numpy labels that later get converted to torch in `collate_fn`